osversion_windows.go

```diff
@@ -1,23 +1,34 @@
 package osversion
 
 import (
-	"errors"
 	"fmt"
-	"syscall"
+	"os/exec"
+	"regexp"
+)
+
+var (
+	verRegex = regexp.MustCompile(`\[Version ([0-9]+\.[0-9]+\.[0-9]+)\]`)
```

Comment:
On my Win10 in Chinese, `cmd ver` shows `Microsoft Windows [版本 10.0.14393]`. To avoid dealing with localized strings, maybe simply `([0-9]+\.[0-9]+\.[0-9]+)` (assuming the brackets may be replaced by something else in certain language)?